tsconfig.json

```diff
@@ -38,7 +38,9 @@
       "#graph-lib": ["libs/graph-lib/src"],
       "#graph-lib/*": ["libs/graph-lib/src/*"],
       "#types": ["libs/types/src"],
-      "#types/*": ["libs/types/src/*"]
+      "#types/*": ["libs/types/src/*"],
+      "#utils": ["libs/utils/src"],
```

Comment:
This is a new directory to store the common utils liek custom guards or decorators. please let me know if you have a better idea how to separate and keep these.

Reply:
You'll also need to add this path mapping to `jest.config.json`

Reply:
Great catch!